setup.py

```diff
@@ -48,27 +50,27 @@ def get_version() -> str:
     return version
 
 
-def get_required_packages(requirement_files: List[str]) -> List[str]:
+def get_required_packages(requirement_files: list[str]) -> list[str]:
     """Get packages from requirements.txt file.
 
     This function returns list of required packages from requirement files.
 
     Args:
-        requirement_files (List[str]): txt files that contains list of required
+        requirement_files (list[str]): txt files that contains list of required
             packages.
 
     Example:
         >>> get_required_packages(requirement_files=["openvino"])
         ['onnx>=1.8.1', 'networkx~=2.5', 'openvino-dev==2021.4.1', ...]
 
     Returns:
-        List[str]: List of required packages
+        list[str]: List of required packages
     """
 
-    required_packages: List[str] = []
+    required_packages: list[str] = []
 
     for requirement_file in requirement_files:
-        with open(f"requirements/{requirement_file}.txt", "r", encoding="utf8") as file:
+        with open(f"requirements/{requirement_file}.txt", encoding="utf8") as file:
```

Comment:
Why this change?

Reply:
`refurb` suggests this change.

`[FURB101]: Use `y = Path(x).read_text()` instead of `with open(x, ...) as f: y = f.read()``

Reply:
This is an automatic change by `pyupgrade`, meaning that we don't need `"r"` in recent versions of python.